src/utils/parsers/currency.js

```diff
@@ -0,0 +1,29 @@
+const REAL_TO_CENTS_FACTOR = 100;
+
+export const centsToReal = (cents) => {
+  if (typeof cents !== 'number' || !Number.isInteger(cents)) {
+    return 'Not valid amount';
+  }
+
+  return (cents / REAL_TO_CENTS_FACTOR).toLocaleString('pt-br', {
+    style: 'currency',
+    currency: 'BRL',
+  });
+};
+
+export const realToCents = (real) => {
+  if (typeof real !== 'string') {
+    throw new TypeError('real must be an string');
```

Comment:
I think we should try to not throw an error from this function as well, for the same reasons as this comment: https://github.com/comarev/comarev-dashboard/pull/80#discussion_r866305831

In my opinion, we should try to avoid throwing errors from util/helper functions, so that we won't have to add lots of try-catches into the components